hadoop-ozone/interface-client/src/main/proto/OmClientProtocol.proto

```diff
@@ -101,7 +101,8 @@ enum Type {
 
   RevokeS3Secret = 93;
 
-  PurgeDirectories = 94;
+  PurgePaths = 94; // [deprecated = true]
```

Comment:
@nandakumar131 I think, below test case will be happy, if we revert `case PurgePaths:` in OmUtils.java.

```
Error:  org.apache.hadoop.ozone.om.ratis.TestOzoneManagerRatisServer.testIsReadOnlyCapturesAllCmdTypeEnums  Time elapsed: 0.693 s  <<< FAILURE!
java.lang.AssertionError: PurgePaths is not categorized in OmUtils#isReadyOnly
	at org.junit.Assert.fail(Assert.java:89)
	at org.junit.Assert.assertTrue(Assert.java:42)
	at org.junit.Assert.assertFalse(Assert.java:65)
	at org.apache.hadoop.ozone.om.ratis.TestOzoneManagerRatisServer.testIsReadOnlyCapturesAllCmdTypeEnums(TestOzoneManagerRatisServer.java:194)
```

Reply:
Fixed it. Thanks @rakeshadr.